index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+    
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+             
+        <a href="./vectors/luis-logo.png">
+            <img class="my-logo" src="./vectors/My Logo.png" alt="Mi logo">
+        </a>
+        <nav>
+            <button class="button">
+                <img src="./vectors/Union.png" alt="nav button">
+            </button>
+            
+        </nav>
+    </header>
+    <main class="main">
+   
+        <div class="headline">
+        
+            <h1 id="principal-title" class="color-blue fonts">I’m Luis Emilio. <br> Glad to see <br> you!
+            </h1>
+            <p class="p-headline fonts">I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded, don’t
+                hesitate to contact me.</p>
+            <p class="linkheadline fonts">LET’S CONNECT</p>
+        
+        
+            <ul class="social-icons-set">
+                <li>
+                    <a href="https://twitter.com/lerfast"><img src="./vectors/twitter Vector.svg" alt="twitter-icon"></a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/luis-emilio-rojas-rojas-6ba90a8a/"><img
+                            src="./vectors/Linkedin icon (1).png" alt="linkedin-icon"></a>
+                </li>
+                <li>
+                    <a href="https://medium.com/"><img src="./vectors/middle.png" alt="medium-icon"></a>
+                </li>
+                <li>
+                    <a href="https://github.com/lerfast"><img src="./vectors/github logo.png" alt="github-icon"></a>
+                </li>
+                <li>
+                    <a href="https://www.angellist.com/"><img src="./vectors/angellist-icon.png" alt="angellist-icon"></a>
+                </li>
+        
+            </ul>
+        </div>
```

Comment:
- [OPTIONAL] Kindly make sure to wrap this container into a `section` tag because It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.
   
   _I believe the previous reviewer made a mistake by suggesting you use `div` instead of using `section` tag because by using `section` you are following HTML best practices and it will be helpful for screen readers_